components/google-cloud/storage/download/component.yaml

```diff
@@ -1,8 +1,8 @@
 name: Download from GCS
 inputs:
-- {name: GCS path, type: URI}
+- {name: GCS path, type: String}
```

Comment:
This change makes the component incompatible with typed component outputs or pipeline parameters.